task-sdk/src/airflow/sdk/execution_time/supervisor.py

```diff
@@ -371,6 +371,9 @@ class WatchedSubprocess:
 
     log: FilteringBoundLogger
 
+    verbose_logs_to_main_logger: bool = False
```

Comment:
the verbosity word here is probably confusing.  verbosity connotes log level.  but this is just about echoing to stdout, or teeing to stdout, irrespective of log level

Reply:
mmm yes good shout. Is `subprocess_logs_to_stdout` better?